modules/virt-creating-infiniband-interface-on-nodes.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-infiniband-interface-on-nodes_{context}"]
+= Creating an IP over InfiniBand interface on nodes
+
+On the {product-title} web console, you can install the NVIDIA Network Operator. The NVIDIA Network Operator supports an InfiniBand (IPoIB) mode. You can use this Operator with other vendor infrastructure to manage NVIDIA resources in a {product-title} cluster. To run  articial intelligence workloads that use NVIDIA resources on your cluster, you must define an InfiniBand (IPoIB) interface in `NodeNetworkConfigurationPolicy` (NNCP) manifest file.
+
+[IMPORTANT]
+====
+The {product-title} documentation describes defining only the IPoIB interface configuration in an `NodeNetworkConfigurationPolicy` (NNCP) manifest file. You must refer to the NVIDIA and other third-party vendor documentation for the majority of the configuring steps. Red{nbsp}Hat support does not extend to anything external to the NNCP configuration.
+====
+
+.Prerequisites
+
+* You have access to specialized hardware.
+* You installed the NVIDIA Network Operator. See link:https://docs.nvidia.com/networking/display/kubernetes2410/getting+started+with+red+hat+openshift[Getting Started with Red{nbsp}Hat OpenShift] (NVIDIA Docs Hub)
+* You installed the CSI driver for DataDirect Networks (DDN) EXAScaler storage. You can find this containerized application on the link:[https://catalog.redhat.com/Red{nbsp}Hat Ecosystem Catalog] website.
```

Comment:
I'm not sure I want to include this. It's very specific to the use case that prompted the bug, but is not generally relevant for IPoIB AFAIK.

In general, I would prefer to just have the "IMPORTANT" note above and leave off the rest of this. Infiniband is not specific to NVidia and I don't want to have to enumerate all of the possible infiniband vendors here.